src/composables/use-load-more.js

```diff
@@ -28,7 +28,7 @@ export const useLoadMore = (props) => {
   })
 
   const onLoadMore = async () => {
-    if (canLoadMore) {
+    if (canLoadMore.value) {
```

Comment:
Oh no! 🙈 
I wish there was a way of preventing this error (not adding `.value` to a check of a ref/computed). I have spent a couple of hours of my life debugging because of this.